biogeochem/EDCohortDynamicsMod.F90

```diff
@@ -307,26 +422,23 @@ subroutine nan_cohort(cc_p)
     currentCohort%c_area             = nan ! areal extent of canopy (m2)
```

Comment:
is crown area not part of PARTEH?

Reply:
PARTEH's state variables are currently all masses.  I did play around with adding an "auxilary" data structure that contained other stuff.  I suppose it has not turned up in PARTEH because crown area does not really play a role in allocation or turnover, it seems to really only turn up in resource acquisition (radiation, canopy layering), stuff outside parteh.